Makefile.ide

```diff
@@ -19,6 +19,15 @@ IDEFILES=$(wildcard ide/*.lang) ide/coq_style.xml ide/coq.png) $(IDEBINDINGS)
 
 .PHONY: coqide
 
+ALL_IDE_FILES=$(shell find ide/coqide -name '*.ml' -or -name '*.mli' -or -name '*.c' -or -name '*.h')
+PROTO_IDE_FILES=$(shell find ide/coqide/protocol -name '*.ml' -or -name '*.mli' -or -name '*.c' -or -name '*.h')
```

Comment:
What about  mll?

Reply:
Fixed thanks.

Reply:
why not `-not -name dune` btw?

Reply:
Actually if the dune file changes we should rebuild too O_O ; I didn't include the other files as they seem runtime deps, not build time deps. Let me actually refine the expression a bit more.

This is a feature that we'd like to see in dune 3, so you can have build deps, and runtime deps that are updated when the command is `(run ...)`ed.

Reply:
I tried to tweak the expression but actually I think all build-time files are already there.